stdlib/public/core/While.swift.gyb

```diff
@@ -0,0 +1,395 @@
+//===--- While.swift.gyb - Lazy views for prefix(while:) & drop(while:) ---===//
```

Comment:
Do they share any implementation details?  If not in any significant way, could you split the file into two, `DropWhile.swift.gyb` and `PrefixWhile.swift.gyb`?